src/frameworks/next/index.ts

```diff
@@ -126,6 +126,22 @@ export async function build(
 
   const env = { ...process.env };
 
+  // Check if the .env.<PROJECT-ID> file exists and make it available for the build process
+  if (context?.projectId) {
+    const envFile = join(dir, `.env.${context.projectId}`);
+
+    if (await pathExists(envFile)) {
+      const envConfig = await readFile(envFile);
+      const envConfigString = envConfig.toString();
+      const envConfigLines = envConfigString.split("\n");
```

Comment:
I worry this is a bit naive of a parser for .env, we need to make sure that we handle escaping and multiline files... maybe we should see if there's some reputable NPM libs?

Reply:
It turns out there's actually a function available ([parseStrict](https://github.com/firebase/firebase-tools/blob/chalosalvador/dotenv-files/src/functions/env.ts#L192)) that parses .env files in a more accurate way.